modules/theme/src/themes/default/assets/images/icons/graph-icon.svg

```diff
@@ -0,0 +1,30 @@
+<!--
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg version="1.1" id="graph-icon" class="icon" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="14" height="14" viewBox="0 0 511.9 511.9" xml:space="preserve">
+	<g>
+		<g>
+			<path class="path" d="M460.8,25.55c-28.2-0.1-51.1,22.8-51.2,51c0,13.3,5.1,26.1,14.4,35.6l-93.5,187.1c-2-0.3-4.1-0.6-6.2-0.6
+				c-6.6,0-13.2,1.3-19.3,3.8l-72.6-81.7c4.2-7.5,6.5-15.9,6.6-24.6c0-28.3-22.9-51.2-51.2-51.2s-51.2,22.9-51.2,51.2
```

Comment:
Please check if the tab spacing is correct.

Reply:
This might be due to the SVG rendering in Github. The formatting looks fine in the IDE.